src/python/ensembl/io/genomio/assembly/download.py

```diff
@@ -277,10 +270,35 @@ def retrieve_assembly_data(
 def main() -> None:
     """Module's entry-point."""
     parser = ArgumentParser(description="Download an assembly data files from INSDC or RefSeq.")
+    parser.add_argument("-v", "--verbose", action="store_true", required=False,
+        help="Verbose level logging")
+    parser.add_argument("-d", "--debug", action="store_true", required=False,
+        help="Debug level logging")
     parser.add_argument("--accession", required=True, help="Genome assembly accession")
     parser.add_argument_dst_path(
         "--download_dir", default=Path.cwd(), help="Folder where the data will be downloaded"
     )
     args = parser.parse_args()
 
-    retrieve_assembly_data(**vars(args))
+    # Configure logging
+    log_file = ("assembly_download.log")
+    date_format='%m/%d/%Y_%I:%M:%S(%p)'
+    logging_format='%(asctime)s - %(levelname)s - %(message)s'
+    reload(logging)
+    if args.verbose:
+        logging.basicConfig(
+            filename=log_file, format=logging_format, datefmt=date_format,
+            filemode="w", level=logging.INFO
+            )
+    elif args.debug:
+        logging.basicConfig(
+            filename=log_file, format=logging_format, datefmt=date_format,
+            filemode="w", level=logging.DEBUG
+            )
+    else:
+        logging.basicConfig(
+            filename=log_file, format=logging_format, datefmt=date_format,
+            filemode="w", level=logging.WARNING
+            )
```

Comment:
We can factorize this further with something like:
```suggestion
    log_level = None
    if args.debug:
           log_level = logging.DEBUG
    elif args.verbose:
           log_level = logging.INFO
    
    logging.basicConfig(
        filename=log_file, format=logging_format, datefmt=date_format,
        filemode="w", level=log_level
        )
```
Note that I've put debug before verbose: assuming we get the 2 params, I suppose debug should take priority.

Reply:
done

Reply:
This is actually better handled by the `add_mutually_exclusive_group()` when adding the parameters to the parser [[link](https://docs.python.org/3/library/argparse.html#mutual-exclusion)].